storage_service/locations/api/resources.py

```diff
@@ -670,7 +671,12 @@ def pointer_file_request(self, request, bundle, **kwargs):
 
     @_custom_endpoint(expected_methods=['get'])
     def check_fixity_request(self, request, bundle, **kwargs):
-        success, failures, message = bundle.obj.check_fixity()
+        try:
+            success, failures, message = bundle.obj.check_fixity()
+        except:
+            success = False
+            failures = []
+            message = 'Internal error'
```

Comment:
Probably good to a) log the error, b) include some additional information in the response. We don't need to expose too much internal app state in the API response, but it'd be good to get an idea *why* it didn't work.